build/Targets/Imports.targets

```diff
@@ -35,6 +35,11 @@
     <_NETCoreSdkIsPreview>false</_NETCoreSdkIsPreview>
   </PropertyGroup>
 
+  <!-- Settings for Visual Studio packages -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.SDK.Analyzers" Version="$(MicrosoftVisualStudioSDKAnalyzersVersion)" PrivateAssets="all" Condition="'$(RoslynProjectType)' == 'Vsix'" />
```

Comment:
All of our shipping Roslyn code lives within non-VSIX projects, so this won't fire there. Was this the condition you intended?

Reply:
➡️ For now, yes. I tried expanding the condition but the change quickly gets out of hand.

Reply:
I don't think we should expand the condition. I think we should remove this entirely. Apply the PackageReference directly to the VSIX files and then add more as time goes along and we find uses for it.

There is no condition which will allow us to apply this at a global location to all of the projects that it needs to be applied to. Hence our end state is going to be what I mention above. May as well start working towards that now. 

---
In reply to: [188648056](https://github.com/dotnet/roslyn/pull/26787#discussion_r188648056) [](ancestors = 188648056)

Reply:
@jaredpar Yes, but that's a separate discussion thread. 😄